.github/workflows/zips.yml

```diff
@@ -232,12 +233,10 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: zip-files
-          path: ${{ github.ref }}
+          path: /tmp
       - name: Extract
         run: |
-          unzip google-site-kit.zip
-          mv google-site-kit/* .
-          rm -rf google-site-kit ./*.zip
+          unzip /tmp/google-site-kit.zip -d ${{ github.ref }}
           echo .distignore > .distignore
```

Comment:
By default `unzip` extracts the contents to the current directory, which would be correct (i.e. same directory as `.distignore` is written). So `-d` would only be needed if for example `/tmp` was the current directory. We could also use it, but target the current directory, or `github.workspace` which should point to the same path. Since we want it in the current directory though, we can just omit that flag (I realize the IB called for it though).

<details>
<summary>man for <code>unzip -d</code></summary>

```
       [-d exdir]
              An optional directory to which to extract files.  By default, all files and subdirectories are recreated in the cur‐
              rent  directory;  the  -d  option allows extraction in an arbitrary directory (always assuming one has permission to
              write to the directory).  This option need not appear at the end of the command line; it is also accepted before the
              zipfile specification (with the normal options), immediately after the zipfile specification, or between the file(s)
              and the -x option.  The option and directory may be concatenated without any white space between them, but note that
              this may cause normal shell behavior to be suppressed.  In particular, ``-d ~'' (tilde) is expanded by Unix C shells
              into the name of the user's home directory, but ``-d~'' is treated as a literal subdirectory ``~''  of  the  current
              directory.
```

</details>

Reply:
@aaemnnosttv want me to address this in a new PR as you've merged this one now?

Reply:
Good catch @aaemnnosttv - I should have spotted the mention of the "current folder" in the IB!